keylime/ima/ast.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, data: str):
         _, _, _, _, sig_size = struct.unpack(fmt, self.data[:hdrlen])
 
         if hdrlen + sig_size != len(self.data):
-            raise ParserError("Invalid signature: malformed header")
+            raise ParserError(f"Invalid signature: malformed header ({hdrlen} + {sig_size} != {len(self.data)})")
```

Comment:
nit: not sure that showing "132 != 128" will be helpful

Reply:
This shouldn't happen very often at all. What it was showing in case of the endianess mixup were values like 32768 != 128.